ethereum/contracts/Application.sol

```diff
@@ -4,4 +4,6 @@ pragma solidity >=0.6.2;
 // Application contains methods that all applications must implement
 abstract contract Application {
     function handle(bytes memory _data) public virtual;
+
+    function register(address _bridge) public virtual returns(bool);
```

Comment:
We'll need doc comments for both `handle` and `register` so we can generate nice Solidity docs.

Reply:
https://github.com/Snowfork/polkadot-ethereum/pull/115/commits/beb6cde3e8933817fca0f11e89e4af106c1bffb2